ciao/arch/architecture.md

```diff
@@ -0,0 +1,423 @@
+---
+title: The Architecture of the Cloud Integrated Advanced Orchestrator (CIAO)
+permalink: architecture.html
+keywords: CIAO, architecture
+---
+
+## A whirlwind tour of the Cloud Integrated Advanced Orchestrator
+
+The Cloud Integrated Advanced Orchestrator (CIAO) is designed to be a simple
+IAAS solution for creating private clouds within an enterprise.  A CIAO cluster
+is composed of a number of different computing devices referred to as nodes.  It
+can be used to deploy units of computation, known as instances, across its
+nodes.  Once deployed, instances can be monitored and managed.  Two types of
+instances are supported; VM based instances, in which all computation is
+performed inside a hypervisor, and container instances, in which docker
+containers are run directly on the cluster nodes.
```

Comment:
Maybe useful to highlight that we also support special "host privileged" containers (privileged non isolated containers)